### DISCUSSION TITLE
[Feature] Support service group in the trace page

### DISCUSSION THREAD

**Discussion by elmerzeng** [apache/skywalking#11695] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

<img width="1132" alt="image" src="https://github.com/apache/skywalking/assets/8157660/1f309225-8809-4bae-8cae-0157862f5c4d">

### Use case

Sometimes, the traces are related some services across group. Is it possible to query traces which are in one group.

### Are you willing to submit a pull request to implement this on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
The query doesn't support to use of multiple service names. If you are proposing this, I can't see how to make it works. `in` is a very slow operation, it is not suitable to query large scale data like traces.

**Reply by elmerzeng** [reply]
All the services in one group could be searched by prefix query. Is it possible to support this function?  We only support one group query one times.

**Reply by wu-sheng** [reply]
No, the query protocol doesn't support that.

**Reply by wu-sheng** [reply]
Query protocol is the thing for the boundaries among UI, backend, and CLI. It can't be changed randomly.

This is Query Protocol through GraphQL. https://github.com/apache/skywalking-query-protocol/blob/master/trace.graphqls

**Reply by elmerzeng** [reply]
Because the trace query support paging search. If we search multiple services in one group one by one, we could support this function.

**Reply by wu-sheng** [reply]
But the reality is, it isn't. Paging and conditions are locked from he protocol perspective. We can't just change it and break everybody else.
